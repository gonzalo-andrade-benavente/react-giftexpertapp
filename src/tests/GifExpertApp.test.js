import { shallow } from "enzyme";
import GifExpertApp from '../GifExpertaApp';


describe( 'Test GifExpertApp.test.js', () => {

    test( 'Should be show the Component Correctly', () => {

        const wrapper = shallow( <GifExpertApp />);
        expect(wrapper).toMatchSnapshot();

    });

});