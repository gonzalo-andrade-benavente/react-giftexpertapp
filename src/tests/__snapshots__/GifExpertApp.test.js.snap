// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test GifExpertApp.test.js Should be show the Component Correctly 1`] = `
<Fragment>
  <h2>
     GiftExpertApp 
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GiftGrid
      category="Naruto"
      key="Naruto"
    />
  </ol>
</Fragment>
`;
